C#/Symbol List Classes.tmPreferences

```diff
@@ -4,11 +4,21 @@
 	<key>name</key>
 	<string>Symbol List: Classes</string>
 	<key>scope</key>
-	<string>source.cs meta.class.identifier</string>
+	<string>source.cs entity.name.class</string>
 	<key>settings</key>
 	<dict>
 		<key>showInSymbolList</key>
 		<integer>1</integer>
+		<key>showInIndexedSymbolList</key>
+		<string>1</string>
+		<key>symbolTransformation</key>
+		<string>
+		s/^/class /;
+		</string>
+		<key>symbolIndexTransformation</key>
+		<string>
+		s/^/class /;
```

Comment:
It seems that you're right, thanks @FichteFoll - when I tested this I must have not waited for my project to be reindexed, because it seemed to work. I must admit that I thought the transformations were only used for display purposes, not lookup purposes as well. But I've proved it with `window.lookup_symbol_in_index` and at the moment, it doesn't find the class name alone, but does find `class ClassName`... I'll make another PR to remove the `symbolIndexTransformation`s.